### ISSUE TITLE
The setting "ticks_on_borders" for vsliders and hsliders refuses to be disabled after Godot freezes while refreshing a related asset.

### ISSUE THREAD

**Issue by ax95** [godotengine/godot#33677]
**Godot version:**
3.1.1 Mono

**OS/device including version:**
Windows 8.1 64

**Issue description:**
While I was overwriting a 'tick' texture for my hslider control (by saving over it in the assets folder through Paint.NET), Godot froze and so I force quit it through task manager. Since then, whenever I open my project (and other projects) and open a scene with a hslider or vslider in it, their "ticks on borders" setting's text is in black, and becomes blue on highlight. If I click on the text or its checkbox, the console prints "Set ticks_on_borders" but the checkbox itself remains ticked. If I click the 'reset' button (the circular arrow), sometimes it's reset back to being disabled, sometimes it does nothing.

No matter the case, the "ticks on borders" setting is always enabled when I run my game, even if they're disabled in the editor. If I close Godot, go to my scene file and manually add "ticks_on_borders = false" underneath my hslider control, that line will be removed once the scene is loaded by Godot. Upon further inspection, it seems like Godot only cares if the line "ticks_on_borders = true" is there or not, and if that line is absent, it is treated like it's disabled. However, in-game and in editor once the scene is loaded, it is not disabled.

If I create a new empty scene with a control root and a hslider control as a child, it functions as expected, but only until I save the scene and quit Godot and then reopen it. Once that is done, the bug appears once more. Reinstalling Godot fixed the issue with other projects, but not for the project I had open once this started happening. I don't know what caused Godot to think "overwrite this setting at all costs" once it froze, but it is quite a nuisance and shouldn't be possible.

Obviously, I expect this and every other setting in a scene to persist once changed inside the editor, in every scene inside a project, for the editor's controls to respond when interacted with, and for all settings of a scene to be properly saved and parsed by Godot. 

**Edit:** I completely removed the Godot folder from my AppData/Roaming folder, and the problem persists. Where else is Godot storing its data? Has this bug always been present and I've only noticed it now?

**Edit 2:** Removing the .mono folder doesn't seem to do anything either. Strangely enough, when I created a new project after a completely fresh reinstall, the hslider I placed down in a new scene displayed this bug, but it disappeared after I saved the scene and then reopened the project. Upon doing so a few times, it wouldn't return. On the other hand, one of my scenes in my main folder now suddenly doesn't display this bug, but one other scene does.

I am just going to assume reading and loading this setting in particular is simply broken, and I'll hope for it to be fixed in a Godot 3.2 stable release.

**Steps to reproduce:**
Place a 'tick' texture for a hslider in your asset folder, and then while you're configuring sliders in the theme editor, just keep overwriting that texture outside of Godot until Godot freezes, I guess. I don't quite know how to reproduce this problem since I don't know what caused it in the first place.

**Minimal reproduction project:**
N/A

**Comment by KoBeWi** [comment]
But did you overwrite the texture again? If the modified time is changed, Godot should reimport it (hopefully correctly this time).

**Comment by ax95** [comment]
I've since solved this problem by resetting the 'show ticks on border' setting on each faulty slider control (with the circular arrow) - despite if it's on or off, since it couldn't be changed - and then saving the scene and relaunching Godot. After doing so. I could no longer reproduce the bug.

I did not try re-importing the texture because I didn't connect the two at the time. It still seems a little strange, but hardly the problem I thought it was - and since I haven't encountered anything like it again nor been able to reproduce this again, it's clearly only on my part, so I'm closing this issue.

Thanks for trying to help, KoBeWi.
